openmc/mgxs/mgxs.py

```diff
@@ -1530,6 +1538,31 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
         df.sort_values(by=[self.domain_type] + columns, inplace=True)
         return df
 
+    def get_units(self, xs_type='macro'):
+        r"""Returns the units of a MGXS.
```

Comment:
Neglected to mention this yesterday, but the "r" isn't necessary to format this docstring since it doesn't include any math (it will still work, but the convention elsewhere has been to format the strings only when necessary). Also, the comment can be shortened to simply "This method returns the units of a MGXS based on a desired xs_type."